contracts/treasury/ArbConnector.sol

```diff
@@ -0,0 +1,80 @@
+//SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import {IGatewayRouter} from "../interfaces/IGatewayRouter.sol";
+
+interface IArbUnionWrapper {
```

Comment:
Create a separate file for `IArbUnionWrapper`, and have `ArbUnionWrapper` to implement it.